components/PopularList.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import styled from "styled-components";
+import { StyledCardImage } from "./StyledCardImage";
+import useSWR from "swr";
+
+const Container = styled.div`
+  display: grid;
+  gap: 10px;
+  grid-template-columns: repeat(auto-fit, minmax(140px, 1fr));
+`;
+
+const fetcher = (...args) => fetch(...args).then((res) => res.json());
+
+function PopularList() {
+  const {
+    data: movies,
+    isLoading,
+    isError,
+  } = useSWR("/api/popular/api_popular", fetcher);
```

Comment:
If you rename the file to `index.js`:
```suggestion
  } = useSWR("/api/popular", fetcher);
```